src/env/mod.rs

```diff
@@ -2,14 +2,35 @@ use crate::api::customization::Customization;
 use crate::api::firmware_protection::FirmwareProtection;
 use crate::api::upgrade_storage::UpgradeStorage;
 use crate::ctap::status_code::Ctap2StatusCode;
-use crate::ctap::Channel;
+use crate::ctap::{Channel, Transport};
 use persistent_store::{Storage, Store};
 use rng256::Rng256;
 
+#[cfg(feature = "std")]
+pub mod host;
 #[cfg(feature = "std")]
 pub mod test;
 pub mod tock;
 
+pub enum SendOrRecvStatus {
```

Comment:
I know this is our own code, but since you touch it anyway, it might make sense to improve! Having a `Result` here could be more idiomatic.

Reply:
Something like this?
```
pub enum SendOrRecvStatus {
    TimedOut,
    Sent,
    Received,
}

pub struct SendOrRecvError;

pub type SendOrRecvResult = Result<SendOrRecvStatus, SendOrRecvError>;
```
Or timeout is better viewed as some kind of error?

Reply:
What you described looks good to me (seeing timeout as a success result).